sdcm/cluster.py

```diff
@@ -40,6 +40,7 @@
 from . import wait
 from .utils import log_run_info, retrying, get_data_dir_path, Distro, verify_scylla_repo_file, S3Storage
 from .loader import CassandraStressExporterSetup
+from .db_stats import PrometheusDBStats
```

Comment:
make sure that you run `microbenchmarking.py --help` when you add new relative imports

Reply:
Ok. Checked